Domains/4-Application/Connectors/ConceptStation/CSPhysical.ecschema.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- ==================================================================================
 |  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-|  * See LICENSE.md in the project root for license terms and full copyright notice.
-======================================================================================= -->
-<ECSchema schemaName="CSSegments" alias="csseg" version="01.00.00" description="iModel Bridge schema for segment data from the ConceptStation application." xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
+|  * See LICENSE.md in the project root for license terms and full copyright notice. ======================================================================================= -->
+<ECSchema schemaName="CSPhysical" alias="csp" version="01.00.00" description="iModel Connector schema for segment data from the ConceptStation application."
```

Comment:
Since the scope of this schema is not only about segments anymore, the description should be tweaked too. e.g. "iModel Connector schema capturing application data from ConceptStation"

Reply:
I agree. I will make the change.